bucket/clash-for-windows.json

```diff
@@ -0,0 +1,20 @@
+{
+    "homepage": "https://docs.cfw.lbyczf.com",
+    "description": "A Windows GUI based on Clash.",
+    "license": "custom",
+    "version": "0.7.11",
+    "url": "https://github.com/Fndroid/clash_for_windows_pkg/releases/download/0.7.11/Clash.for.Windows-0.7.11-win.7z",
+    "hash": "3fb9a5d2829b10506b407b335b78eb0ee6298de83765c6f6a037a8dcb890d175",
+    "bin": "Clash for Windows.exe",
+    "shortcuts": [
+        [
+            "Clash for Windows.exe",
+            "Clash for Windows"
+        ]
+    ],
+    "checkver": "github",
+    "autoupdate": {
+        "url": "https://github.com/Fndroid/clash_for_windows_pkg/releases/download/$version/Clash.for.Windows-$version-win.7z",
+        "hash": "$baseurl/sha256sum"
```

Comment:
...

```suggestion
        "hash": {
            "url": "$baseurl/sha256sum",
            "regex": "7z:\\s+$sha256"
        }
```

Reply:
the latest version have received this change.